fix(*): "We"/"our"/etc. to "you"/"your"/etc. (#661)
discordjs/guide: guide/voice/optimisation-and-troubleshooting.md

#### From YouTube videos

> Many voice bots allow an ability to play audio from YouTube videos in voice channels. YouTube itself provides WebM/Ogg streams for newer videos, and so you can also demux these files for Opus audio instead of running them through an FFmpeg transcoder first.

Reviewer: ```suggestion
Many voice bots have the ability to play audio from YouTube videos in voice channels. YouTube itself provides WebM/Ogg streams for newer videos, which you can demux for Opus audio instead of running them through an FFmpeg transcoder first.
```

```

::: tip

> You might be wondering why the type is `opus` and not `webm/opus` or `ogg/opus`. discord.js allows us to play Opus streams **without a container** operating in object-mode (i.e., each item pushed to the stream is a distinct Opus packet). `ytdl-core-discord` provides this type of stream, and so you must specify `opus` as the type.

Reviewer: ```suggestion
You might be wondering why the type is `opus` and not `webm/opus` or `ogg/opus`. Discord.js allows you to play Opus streams **without a container** operating in object-mode (i.e., each item pushed to the stream is a distinct Opus packet). `ytdl-core-discord` provides this type of stream, and so you must specify `opus` as the type.
```
